test/org/zaproxy/zap/extension/pscanrulesAlpha/XAspNetVersionScannerUnitTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2017 The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.parosproxy.paros.core.scanner.Plugin.AlertThreshold;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.pscanrulesAlpha.XAspNetVersionScanner;
+
+public class XAspNetVersionScannerUnitTest extends PassiveScannerTest {
+
+    @Override
+    protected XAspNetVersionScanner createScanner() {
+        return new XAspNetVersionScanner();
+    }
+
+    @Test
+    public void shouldRaiseAlertWhenResponseContainsXAspNetVersionHeader() throws HttpMalformedHeaderException {
+
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET http://www.example.com/test/ HTTP/1.1");
+
+        msg.setResponseBody("<html></html>");
```

Comment:
This (and in following test) could be removed, it's not relevant to the tests (i.e. the scanner is not checking the contents of the message while this seems to imply that it needs to have something).